homeassistant/components/steam_online/config_flow.py

```diff
@@ -108,6 +108,11 @@ async def async_step_reauth_confirm(
         )
 
 
+def _batch_ids(ids: list[str]) -> Generator:
```

Comment:
You can add the Generator YieldType here with `Generator[str, None, None]`, or, if it's not important that the type be known as a generator, you can use `Iterator[str]` instead. I think Iterator is fine here because you're using this to create a list anyway.

https://docs.python.org/3.11/library/typing.html#typing.Generator